cmd/statusd/library.go

```diff
@@ -307,6 +308,24 @@ func Call(chatID *C.char, path *C.char, params *C.char) *C.char {
 	return C.CString(res)
 }
 
+//export StartCPUProfile
+func StartCPUProfile(path *C.char) *C.char {
+	err := profiling.StartCPUProfile(C.GoString(path))
+	return makeJSONResponse(err)
+}
+
+//export StopCPUProfiling
+func StopCPUProfiling() *C.char {
+	err := profiling.StopCPUProfile()
+	return makeJSONResponse(err)
+}
+
+//export WriteHeapProfile
+func WriteHeapProfile(path *C.char) *C.char {
+	err := profiling.WriteHeapFile(C.GoString(path))
+	return makeJSONResponse(err)
+}
```

Comment:
Can you, please, update https://github.com/status-im/status-go/wiki/Notes-on-Bindings according to the new changes?